tiatoolbox/models/__init__.py

```diff
@@ -20,3 +20,17 @@
 
 """Models package for the models implemented in tiatoolbox."""
 from tiatoolbox.models import abc, architecture, dataset
+from tiatoolbox.models.controller.nucleus_instance_segmentor import (
```

Comment:
shouldn't we go with `tiatoolbox.models.engin` instead of `controller` in this PR?

Reply:
That requires changing everything now too btw. Srijay is doing that atm for `develope`. Either we merge this one first and deal with the renaming later, or we stall till Srijay PR gets merged and then I deal with this.

@shaneahmed what do you think?

Reply:
Let's wait for @Srijay-lab PR. I think his PR should be quick to merge.